raijin_scripts/execute_ingest/run

```diff
@@ -91,5 +91,5 @@ echo "Executing ingest for ${YEAR} ${PRODUCT}"
 ##################################################################################################
 # Run dea-submit-ingest process
 ##################################################################################################
-yes Y | dea-submit-ingest qsub --project "${PROJECT}" --queue "${QUEUE}" -n 5 -t 10 --name "${JOB_NAME}" "${PRODUCT}" "${YEAR}"
+yes Y | dea-submit-ingest qsub --project "${PROJECT}" --queue "${QUEUE}" -n 5 -t 40 --name "${JOB_NAME}" "${PRODUCT}" "${YEAR}"
```

Comment:
Let's leave it smaller for now. If we can keep it up to date 5 nodes and 10 hours is far more than required. It's just that we haven't had it run successfully for months.

Reply:
Revert back changes